### DISCUSSION TITLE
Upload media is not working

### DISCUSSION THREAD

**Discussion by Dirki16** [mastodon/mastodon#19497] [Q&A]
Hello,
I have the following problem:
When I try to upload media such as a profile picture, I get the error "We're sorry, but something went wrong on our end." I have set Google Cloud Service to store the media.
I imagine the problem is the connection between my server and GCS. Can someone help me to solve the problem?

**Comment by tkforty** [comment]
I am having the same problem and am using Google Cloud Storage. . . In Sidekiq I see the error "Aws::S3::Errors::InvalidArgument: Invalid argument.". Were you able to resolve this?
